GREEDY.UnitTests/DataManagersUnitTests/UnitTestDataConverter.cs

```diff
@@ -1,82 +1,80 @@
+//            //act
 
-            //assert
-            Assert.True(dataConverter.ReceiptToItemList(receipt).Count == 1);
-            //Assert.Equal("vanduo", dataConverter.ReceiptToItemList(receipt)[0].Name);
-            //Assert.Equal((decimal)5.22, dataConverter.ReceiptToItemList(receipt)[0].Price);
-        }
+//            //assert
+//            Assert.True(dataConverter.ReceiptToItemList(receipt).Count == 1);
+//            //Assert.Equal("vanduo", dataConverter.ReceiptToItemList(receipt)[0].Name);
+//            //Assert.Equal((decimal)5.22, dataConverter.ReceiptToItemList(receipt)[0].Price);
+//        }
 
-        [Fact]
-        public void DataConverter_ReceiptToItemList_IKI_ItemToMatchRegex()
-        {
-            //arrange
-            var fixture = new Fixture();
-            var list = new List<string>
-            {
-                "PALINK\r\r\n" +
-                "LT101937219\r\r\n",
-                "VILKYŠKIU 1,14 A\r\r\n",
-                "BANANAI 0›60 A\r\r\n",
-                "Prekiautojo 10 15057003 \r\r\n"
-            };
+//        [Fact]
+//        public void DataConverter_ReceiptToItemList_IKI_ItemToMatchRegex()
+//        {
+//            //arrange
+//            var fixture = new Fixture();
+//            var list = new List<string>
+//            {
+//                "PALINK\r\r\n" +
+//                "LT101937219\r\r\n",
+//                "VILKYŠKIU 1,14 A\r\r\n",
+//                "BANANAI 0›60 A\r\r\n",
+//                "Prekiautojo 10 15057003 \r\r\n"
+//            };
 
-            fixture.AddManyTo(list);
-            Receipt receipt = fixture.Build<Receipt>().With(x => x.LinesOfText, list).Create();
-            var dataConverter = new DataConverter();
-            //act
+//            fixture.AddManyTo(list);
+//            Receipt receipt = fixture.Build<Receipt>().With(x => x.LinesOfText, list).Create();
+//            var dataConverter = new DataConverter();
+//            //act
 
-            //assert
-            Assert.True(dataConverter.ReceiptToItemList(receipt).Count == 2);
-            //Assert.Equal("VILKYŠKIU", dataConverter.ReceiptToItemList(receipt)[1].Name);
-            //Assert.Equal("BANANAI", dataConverter.ReceiptToItemList(receipt)[1].Name);
-            //Assert.Equal((decimal)1.14, dataConverter.ReceiptToItemList(receipt)[1].Price);
-            //Assert.Equal((decimal)0.60, dataConverter.ReceiptToItemList(receipt)[1].Price);
-        }
-    }
-}
+//            //assert
+//            Assert.True(dataConverter.ReceiptToItemList(receipt).Count == 2);
+//            //Assert.Equal("VILKYŠKIU", dataConverter.ReceiptToItemList(receipt)[1].Name);
+//            //Assert.Equal("BANANAI", dataConverter.ReceiptToItemList(receipt)[1].Name);
+//            //Assert.Equal((decimal)1.14, dataConverter.ReceiptToItemList(receipt)[1].Price);
+//            //Assert.Equal((decimal)0.60, dataConverter.ReceiptToItemList(receipt)[1].Price);
+//        }
+//    }
+//}
```

Comment:
You told me yourself not to leave commented code

Reply:
Sure. I create this pull request just to get your oppinion about main changes. Unit test will be added later